src/region.js

```diff
@@ -232,7 +184,7 @@ var Region = MarionetteObject.extend({
     // present before replacing
     this._restoreEl();
 
-    var parent = this.el.parentNode;
+    const parent = this.el.parentNode;
 
     parent.replaceChild(view.el, this.el);
     this.replaced = true;
```

Comment:
We should probably go ahead and match our styles here and go with `_isReplaced` or something similar..